homework9/my-app/src/components/Header.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import Button from "./Button";
+import Logo from "./Logo";
+import SiteNav from "./SiteNav";
```

Comment:
try to sort your imports.
In first group should be impors from `npm`
then empty line
second group - imports your `reusable components, variables, functions, e.t.c `(for that case it is from components folder)
empty line
then your local imports `that exist only in that folder`
empty line
import `styles`
empty line

It make your code easier for reading

empty line
then your local imports that exist only in that folder
empty line
import styles
empty line

It makes your code easier for reading

Reply:
example:

import React from "react";

import Button from "./Button";
import SiteNav from "./SiteNav"; 
import Logo from "./Logo";

import './styles.css'

Reply:
also you can create `jsconfig.json` flie adn add there some configuration, for example for default path and then path for your reusable component will be like `components/Button` instead of `../../../../../components/Button `- it not case from your project